Source/RxComponents/LMCDampedHarmonicOscillator.cc

```diff
@@ -0,0 +1,95 @@
+/*
+ * LMCDampedHarmonicOscillator.cc
+ *
+ *  Created on: Jul 6, 2022
+ *      Author: pslocum
+ */
+
+#include "LMCDampedHarmonicOscillator.hh"
+#include "logger.hh"
+
+namespace locust
+{
+
+	LOGGER( lmclog, "DampedHarmonicOscillator" );
+
+    DampedHarmonicOscillator::DampedHarmonicOscillator():
+    		fMaxNBins( 4000 ),
+			fTimeResolution( 1.e-12 ),
+			fCavityFrequency( 1.0e9 ),
+			fCavityQ( 1000 ),
+			fThresholdFactor ( 0.25 ),
```

Comment:
The combination of preset value for `fMaxNBins` doesn't make much sense given the other preset values. For the given parameters, `fMaxNBins` needs to be at least ~4.4e5 to contain everything up the `fThresholdFactor` of `0.25`. The current number of `4000` bins will only go down to `0.9875` of the initial value. 
Also, setting the threshold to 1/4th sounds like it would affect the resulting output by quite a bit.

The need to have such a long impulse response is something that has been concerning me for a while though ☹️

Reply:
Aha thanks, we will be using the debugging FIR.txt file to check this type of issue.  But yes, they are arbitrary values at the moment.